modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingListenMoreScreen.kt

```diff
@@ -4,6 +4,6 @@ import androidx.compose.runtime.Composable
 import au.com.shiftyjelly.pocketcasts.compose.components.TextH10
 
 @Composable
-fun GiveRatingListenMore() {
+fun GiveRatingListenMoreScreen() {
```

Comment:
This class looks redundant after this commit: https://github.com/Automattic/pocket-casts-android/pull/1451/commits/107de822cf4aeb0aef0f04d7847dcf5f5b03c4c2

Reply:
You're right, I forgot to remove that. 👍 Addressed in https://github.com/Automattic/pocket-casts-android/pull/1451/commits/0913b142c79454f31f14643183595d7e84ef85e8